openwisp_radius/api/urls.py

```diff
@@ -11,6 +11,16 @@
     url(r'^registration/(?P<slug>[\w-]+)/$',
         views.register,
         name='rest_register'),
+    # password reset
+    url(r'^password/reset/$',
```

Comment:
here's the first challenge of this project, which I didn't notice before when reviewing, sorry for that.

Do you see how the other URLs have the `(?P<slug>[\w-]+)` parameter?
that's because each of these URLs has to be called by different organizations.

Since the password change and reset operations will be performed from the captive pages of different organizations, we need to include this parameter in the URL and have each view use this parameter to check whether the user belongs to the organizations or not, if not, return something like `403` HTTP status code.

Reply:
Since email is unique to every user, I thought we can let users change the password without verifying the organisation. I don't think there is a need to verify the organisation for resetting the password.

Also, the email field for the users added with batch API view is empty and we don't have any API view to let users edit email or any other information. Those users can't change their password if forgotten. Should I add it to the issues?

Reply:
you are missing the point, these URLs will be called from a captive page that is specific to an organization, so if the user is not member of that organization it makes no sense to let that user reset their password with that organization